README.md

```diff
@@ -14,14 +13,12 @@ from one endpoint to another service.
   - [data-collector](https://github.com/ManageIQ/aiops-data-collector)
   - [publisher](https://github.com/ManageIQ/aiops-publisher)
 * Discover all AI services we're integrating with
-  - [dummy-ai](https://github.com/ManageIQ/aiops-dummy-ai-service)
-  - [aicoe-insights-clustering](https://github.com/RedHatInsights/aicoe-insights-clustering)
+  - [aiops-dummy-ai-service](https://github.com/ManageIQ/aiops-ai-library/tree/dummy)
+  - [aiops-volume-type-validation](https://github.com/ManageIQ/aiops-ai-library/tree/volume_type_validation)
+  - [aiops-outlier-detection](https://github.com/ManageIQ/aiops-ai-library/tree/outlier_detection)
+  - [aiops-insights-clustering](https://github.com/ManageIQ/aiops-insights-clustering)
```

Comment:
And this is technically not a part of `aiops-ai-library` - so include or exclude?

Reply:
It's not part of the library, correct, though it's our usecase and the header here says `Discover all AI services we're integrating with` so I think we can keep it... The idea is to list all the AI services we're integrating here. So.. why not... :)

Reply:
Alright!